site/gatsby-site/src/global.css

```diff
@@ -48,6 +48,21 @@ body {
   font-size: 16px;
 }
 
+/********************************************************** 
+ * This allows us to set `flex-grow` on the main content  *
+ * so that the footer rests at the bottom of the viewport *
+ * if the content is not large enough to fill it.         *
+ **********************************************************/
+html, body, #___gatsby {
+  height: 100%;
+}
+#gatsby-focus-wrapper {
```

Comment:
Since looks like you only need `flex` for the `LayoutHidden` component, can you add that there?

Setting `flex` to every other page seems unnecessary, and it is kind of unexpected.

@clari182 is also working on layouts/menus might have something to add here